.github/workflows/ci.yml

```diff
@@ -25,6 +25,8 @@ jobs:
         npm ci
         npm run generate-component-install
         npm run doc-site-install
+    - name: Check Types
```

Comment:
this is embarrassing, but we never checked our types... (not sure if anyone did it locally too 😅)